src/components/OutputForm.tsx

```diff
@@ -47,17 +48,43 @@ export const OutputText = ({ label, value }: { label: string, value?: string })
   );
 }
 
-export const OutputTextArea = ({ label, value }: { label: string, value?: string }) => {
+export const OutputTextArea = ({ label, value, maxHeight }: { label: string, value?: string, maxHeight?: number }) => {
```

Comment:
How does this change handle maxHeight being set, but the content is shorter than the maxHeight? Seems like the "Show More" button will still be visible, but do nothing?

Reply:
yeah, the problem is we don't actually know how tall the content is. For example, if the description is a 200 word paragraph with no line breaks, the component only has 1 Typography element, but that element may wrap 20 lines. I think for now, the experience is fine -- I'd rather have the button show and not really be needed, versus, not implementing it and having someone enter a giant description that cannot be shrunk.

Maybe at some point we can figure out a way to infer the height of the content by the length or something.

Reply:
looking into this... it might be easier than I thought

Reply:
Gotcha. And ya, looking it up it seems it can get messy to figure that out. Seems like there may be a way using react effects, which allow you to get the height of a rendered element, but agreed with getting this in now and fixing that later.

Reply:
I was able to address this in the commit I just made.